README.md

```diff
@@ -18,16 +18,17 @@ To setup on your local machine:
     ```
     git clone https://github.com/Microsoft/Recommenders
     ```
-3. Run the generate conda file script and create a conda environment:   
+3. Run the generate conda file script to create a conda environment:
+   (This is for a basic python environment, see [SETUP.md](SETUP.md) for PySpark and GPU environment setup) 
     ```
     cd Recommenders
-    ./scripts/generate_conda_file.sh
-    conda env create -n reco -f conda_bare.yaml  
+    python scripts/generate_conda_file.py
+    conda env create -f reco_base.yaml  
     ```
 4. Activate the conda environment and register it with Jupyter:
     ```
-    conda activate reco
-    python -m ipykernel install --user --name reco --display-name "Python (reco)"
+    conda activate reco_base
+    python -m ipykernel install --user --name reco_base --display-name "Python (reco)"
     ```
 5. Start the Jupyter notebook server
     ```
```

Comment:
"6. Run the SAR python CPU movielens" link is broken.

Btw, why SAR notebook? because it is no-spark/no-gpu example, or this is from back in the days when we only have SAR here? If this is an example, we can say more context something like, "to check the env setup, start the jupyter server and open one of our notebooks... "

Reply:
Also I found some typos too. E.g. Data Preparation Notebook shows  --> now we have 2 notebooks ! ;-)

Reply:
Algorithms table has wrong link:
SVD links to **SAR** notebook

Reply:
In the table, not sure about current categorization. xDeepFM and DKN are both deep learning recommenders too but categorized under MS's recommenders. We should better categorization . Do we have an issue created for this? I remember we once discussed about better categorization @miguelgfierro @yueguoguo @anargyri

Reply:
I addressed a couple of the smaller issues
- fixed the SAR python CPU movielens link
- fixed Data Prep typo
- fixed SVD link in algorithms table

Since the others are out of scope for this PR I didn't address them. @loomlike can you create a separate issue for those?